.storybook/preview.js

```diff
@@ -0,0 +1,30 @@
+import IconSettings from '@salesforce/design-system-react/components/icon-settings';
+import actionSprite from '@salesforce-ux/design-system/assets/icons/action-sprite/svg/symbols.svg';
+import customSprite from '@salesforce-ux/design-system/assets/icons/custom-sprite/svg/symbols.svg';
+import doctypeSprite from '@salesforce-ux/design-system/assets/icons/doctype-sprite/svg/symbols.svg';
+import standardSprite from '@salesforce-ux/design-system/assets/icons/standard-sprite/svg/symbols.svg';
+import utilitySprite from '@salesforce-ux/design-system/assets/icons/utility-sprite/svg/symbols.svg';
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+
+import 'sass/app.scss';
+
+export const parameters = {
+  actions: { argTypesRegex: '^on[A-Z].*' },
+};
+
+export const decorators = [
+  (Story) => (
+    <MemoryRouter>
+      <IconSettings
```

Comment:
eek, looks like docs for [Icon Settings](https://react.lightningdesignsystem.com/components/icon-settings/) aren't working but this makes sense!

Reply:
@duggiemitchell Huh, good call -- weird about the docs.

I basically tried to re-create the "wrapping" setup that we're using in `src/js/index.tsx` (https://github.com/oddbird/Metecho/blob/main/src/js/index.tsx#L179-L191) or test files. One thing I didn't worry about is whether this setup will break for a component that tries to access the Redux store. If we want to document such a component, we'll have to test it out and see what the best workaround is (probably a similar approach as our test files, mocking out a minimal Redux store for the specific story).